### PR TITLE
Liquid hooks

### PR DESCRIPTION
**Pull request by thescubageek**
Adds a new optional variable to Garden Widgets  - Liquid - which denotes which widgets will support liquid variables in their fields. This is used in conjunction with changes to the CMS (PR: https://github.com/G5/g5-content-management-system/pull/578/files) to facilitate advanced spinup options from seeder files via Liquid, as well as enable Web Builders to use Liquid variables in select widgets.

So far, this has been enabled for the following widgets:

* HTML
* Meta Description
* Photo

Enabling Liquid for Garden Widgets is as easy as adding the following to the index.html file for the widget:

```html
<dt>Liquid</dt>
<dd><span class="p-liquid">true</span></dd>
```

Also fixes a bunch of typos where Configuration was misspelled as Congfiguration, because I'm OCD like that...

*****

**Update:**
This now includes changes to the Content Stripe and Column widget. These changes do the following:

* Fix bugs where these widgets were generating duplicate IDs in the DOM
* Renames the settings for these widgets to use integers instead of the english-language numbers, ie "row_1_widget_id" vs "row_one_widget_id"

Both of these changes correspond with settings and widgets migrations to the CMS in https://github.com/G5/g5-content-management-system/pull/578

### CODE REVIEW

**Comment by variousred** [comment]
classic-kong1-700x700

**Comment by variousred** [comment]
@thescubageek i'd make the property with a vendor prefix p-g5-liquid

File: public/static/components/button/index.html [hunk 1]
```diff
@@ -9,7 +9,7 @@
         <dd><span class="p-widget-id">4</span></dd>
         <dt>Summary</dt>
         <dd><span class="p-summary">A link styled like a button</span></dd>
-        <dt>Congfiguration</dt>
+        <dt>Configuration</dt>
```
Comment (bbauer) [inline public/static/components/button/index.html]: I prefer "Conflagration"

**Comment by bbauer** [comment]
:+1:  if green

**Pull request merged**